MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/javaservices/TilequeryActivity.java

```diff
@@ -77,13 +78,18 @@ protected void onCreate(Bundle savedInstanceState) {
 
   @SuppressWarnings({"MissingPermission"})
   @Override
-  public void onMapReady(MapboxMap mapboxMap) {
+  public void onMapReady(@NonNull final MapboxMap mapboxMap) {
     TilequeryActivity.this.mapboxMap = mapboxMap;
-    addClickLayer();
-    addResultLayer();
-    displayDeviceLocation();
-    mapboxMap.addOnMapClickListener(this);
-    Toast.makeText(this, R.string.click_on_map_instruction, Toast.LENGTH_SHORT).show();
+    mapboxMap.setStyle(Style.MAPBOX_STREETS, new Style.OnStyleLoaded() {
+      @Override
+      public void onStyleLoaded(@NonNull Style style) {
+        addClickLayer();
+        addResultLayer();
+        displayDeviceLocation();
```

Comment:
provide style as parameter to above methods

Reply:
fixed